Add multiple language support for Standard & Advanced Integration

amsourav opened the pull request:
feat: add backend and frontend implementations
feat: update README.md
feat: added code for standard integration
chore: delete full stack code and update package names

Commit:
Add multiple language support for Standard & Advanced Integration

feat: add backend and frontend implementations
feat: update README.md
feat: added code for standard integration
chore: delete full stack code and update package names

Commit:
feat: update devcontainer for v2

Commit:
feat: update devcontainer for v1

amsourav committed:
Merge pull request #1 from amsourav/feature/devcontainer-support-for-new-lanugage

Feature/devcontainer support for new lanugage

wickerpopstar commented on advanced-integration/v2/backend/dotnet/Server.cs:
```diff
@@ -0,0 +1,191 @@
+using Microsoft.Extensions.Hosting;
+using Newtonsoft.Json;
+
+namespace PayPalAdvancedIntegration
```
Is there a specific version of C#/.NET you're targeting here? I think it's probably more common to use a simple namespace declaration `namespace PayPalAdvancedIntegration;` instead of enclosing the whole file in curly braces.

amsourav replied:
We are targeting v8.0. I'll remove the curly braces

wickerpopstar reviewed (approved):
.NET looks good.

LeonarthusMectus commented on advanced-integration/v2/backend/php/CONTRIBUTING.md:
```diff
@@ -0,0 +1,14 @@
+# How to Contribute
```
It doesn't look like this example uses Slim, was this pulled over from a previous/different example?  We can probably just remove the whole thing.

LeonarthusMectus commented on advanced-integration/v2/backend/php/docker-compose.yml:
```diff
@@ -0,0 +1,18 @@
+version: '3.7'
```
I don't see that other languages have a docker/compose file.  Is this needed?

LeonarthusMectus commented on standard-integration/backend/php/CONTRIBUTING.md:
```diff
@@ -0,0 +1,14 @@
+# How to Contribute
```
Same as advanced example, do we need this?

rekhaabaidipati commented on .devcontainer/advanced-integration-v1/devcontainer.json:
```diff
@@ -6,15 +6,19 @@
   // Use 'postAttachCommand' to run commands when attaching to the container.
   "postAttachCommand": {
-    "Start server": "npm start"
+    "Start server": "cd backend/node && npm start",
```
Why do we need backend and frontend folders . Can we just have client and server folders with respective language folders similar to this repo : https://github.com/paypal-examples/fastlane-sample-application/tree/main/server

amsourav replied:
I can rename them to sever and client

amsourav commented on standard-integration/backend/php/docker-compose.yml:
```diff
@@ -0,0 +1,18 @@
+version: '3.7'
```
I'll remove it

amsourav replied:
Removed

Commit:
chore: Update folder names and remove unused files

- Rename backend, frontend folders to server, client respectively
- Remove SlimPHP contribution guidelines
- Remove docker-compose.yml from php
- Use file scoped namespace for dotnet

Commit:
docs: update docs to use the server/client name convention

rekhaabaidipati reviewed (commented):
Thanks for addressing my feedback. LGTM for dev container changes and overall folder structure.

rekhaabaidipati reviewed (approved):
LGTM
